packages/Tooltip/src/__mocks__/Tooltip.mock.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import '../tooltip.scss';
+
+export type Appearance = 'dark' | 'light';
+export type Position = 'top' | 'right' | 'bottom' | 'left' | 'auto';
+
+export interface TooltipProps extends React.ComponentPropsWithoutRef<'div'> {
+  /** The target button, text, svg etc.. of the Tooltip. */
+  children: React.ReactNode;
+  /** Add a specific class to the tooltip */
+  tooltipClassName?: string;
+  /** The content to display inside the tooltip */
+  content?: React.ReactNode;
+  /** The position the tooltip is on. */
+  position?: Position;
+  /** The appearance of the tooltip. */
+  appearance?: Appearance;
+  /** The max with of the tooltip */
+  maxWidth?: number;
+  /** When True, manually show the Tooltip. */
```

Comment:
To fit with your other doc, it should not end with a dot and you will have to choose between `Tooltip` or `tooltip`